src/backend/distributed/connection/remote_commands.c

```diff
@@ -82,6 +82,51 @@ ForgetResults(MultiConnection *connection)
 
 
 /*
+ * ClearRemainingResults clears a connection from pending activity,
+ * returns true if all pending commands return success. It raises
+ * error if any command fails and transaction is deemed critical.
+ *
+ * Note that this might require network IO. If that's not acceptable, use
+ * NonblockingForgetResults().
+ */
+bool
+ClearRemainingResults(MultiConnection *connection)
```

Comment:
To future-proof this function. What if we add the raiseErrors flag as a parameter?

Reply:
ok